server/db/schema.js

```diff
@@ -32,6 +34,26 @@ const schema = {
         "data" jsonb
         )`,
     },
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.PARTICIPANTS_DISTANCE} (
+        id serial primary key,
+        "participant_id" integer not null,
+        "site_id" integer not null,
+        "distance" integer not null
```

Comment:
I think we should make distance nullable for now until we have a final decision if we should round unexpected postal codes or just leave it empty